index.html

```diff
@@ -430,7 +439,8 @@
       [[FULLSCREEN]].</p>
 
       <p>Pointer lock must be exited if the
-      <a>target</a> is removed from its document, or the <a>user agent</a>,
+      <a>target</a> is removed and is no longer in a shadow-including document,
```

Comment:
I think this should use https://html.spec.whatwg.org/#node-is-disconnected: "Pointer lock must be exited if the target is [disconnected](https://html.spec.whatwg.org/multipage/infrastructure.html#node-is-disconnected), or the user agent, window, or tab loses focus."